docs/architecture/deployment/deployment.md

```diff
@@ -61,15 +61,17 @@ Teams are the primary mechanism of ownership. Teams own Projects where Pipelines
 
 ### SQL
 
+To allow for SQL query execution Prophecy can connect to Snowflake and Databricks warehouses, with connectors for additional SQL warehouses coming soon.
+
 #### Snowflake
 
-To connect with data stored in a SQL Warehouse, or to allow for interactive SQL execution, Prophecy can connect to an existing Snowflake execution environment through a secure and performant **Snowpark** interface.
+To connect with data stored in a SQL Warehouse, or to allow for interactive SQL execution, Prophecy can connect to an existing Snowflake execution environment through secure and performant [Snowpark]](https://docs.snowflake.com/en/developer-guide/snowpark/index) or [Snowflake](https://docs.snowflake.com/en/developer-guide/sql-api/reference) APIs.
 
 Each [Fabric](../../concepts/fabrics) defined in Prophecy connects to a single Snowflake Warehouse and each user is required to provide credentials to authenticate to it.
```

Comment:
"each user is required to provide credentials to authenticate to it."
That's not actually correct, since the fabric is shared among the team members. 

A user could provide creds in a snowflake [source](https://docs.prophecy.io/low-code-spark/gems/source-target/warehouse/snowflake/). Would that be shared as a dataset across the team?